test/Jamfile

```diff
@@ -43,7 +49,7 @@ alias fail_if_no_openssl
 explicit fail_if_no_openssl ;
 
 # Requirements to use across targets
-local requirements = 
+local requirements =
```

Comment:
This is just a question - am I allowed to use the `boost_mysql` target in root `build.jam` in my tests? If it is, I will refactor the test Jamfiles to use it, instead of current `/boost/mysql/test//boost_mysql`.